src/main/java/org/seqdoop/hadoop_bam/SplittingBAMIndexer.java

```diff
@@ -194,6 +194,13 @@ public void processAlignment(final SAMRecord rec) throws IOException {
 		count++;
 	}
 
+	void processAlignment(final long virtualOffset) throws IOException {
```

Comment:
It's used in BAMRecordWriter to write splitting BAM indexes from MR. See https://github.com/HadoopGenomics/Hadoop-BAM/pull/111/commits/cd89366d86302ba7d0c02079fc3d663dd46408ce#diff-8ce0e02bf18a51ccfbbe62aebc2f9498R141.